common/replication/block_fetcher.go

```diff
@@ -316,24 +314,34 @@ type BlockFetcher struct {
 	setupVerifierInProgress bool
 }
 
-func NewBlockFetcher(fc FetcherConfig, signer identity.SignerSerializer, bccsp bccsp.BCCSP, lastConfig channelconfig.Resources, dialerConfig comm.ClientConfig, verifyBlockSequence func(blocks []*common.Block, _ string) error, stopChannel chan struct{}) (*BlockFetcher, error) {
+func NewBlockFetcher(fc FetcherConfig, signer identity.SignerSerializer, bccsp bccsp.BCCSP, lastConfig channelconfig.Resources, dialerConfig comm.ClientConfig, verifyBlockSequence BlockSequenceVerifier, stopChannel chan struct{}) (*BlockFetcher, error) {
```

Comment:
`verifyBlockSequence BlockSequenceVerifier` - so, we inject a verification function to be used by `BlockPuller` even though we have a `verifierFactory` for the `BlockFetcher` ?

Reply:
Instead we should just make the `verifyBlockSequence` be a no-op function.